curriculum/challenges/english/25-front-end-development/lab-polygon-area-calculator/5e444147903586ffb414c94f.md

```diff
@@ -0,0 +1,881 @@
+---
+id: 5e444147903586ffb414c94f
+title: Build a Polygon Area Calculator
+challengeType: 23
```

Comment:
```suggestion
challengeType: 27
```
We have a new challenge type https://github.com/freeCodeCamp/freeCodeCamp/pull/59732